x-pack/plugins/cases/public/components/user_action_tree/user_action_host_isolation_comment_event.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, useCallback } from 'react';
+import * as i18n from './translations';
+import { LinkAnchor } from '../links';
+import { useKibana } from '../../common/lib/kibana';
+import { SECURITY_SOLUTION_APP_ID } from './constants';
+
+interface Props {
+  type: string;
+  hostId: string;
+  hostName: string;
+}
+
+const HostIsolationCommentEventComponent: React.FC<Props> = ({ type, hostId, hostName }) => {
+  const activityLogPath = `/administration/endpoints?selected_endpoint=${hostId}&show=activity_log`;
+  const { getUrlForApp, navigateToUrl } = useKibana().services.application;
+  const endpointDetailsHref = getUrlForApp(SECURITY_SOLUTION_APP_ID, {
+    path: activityLogPath,
+  });
+
+  const onLinkClick = useCallback(
+    (ev) => {
+      ev.preventDefault();
+      return navigateToUrl(endpointDetailsHref);
+    },
+    [endpointDetailsHref, navigateToUrl]
+  );
+
+  return (
```

Comment:
Given the comment I just posted below for the API change, you may need to check to ensure you have a `hostId` and if not, then you should not return a `LinkAnchor` because that will be a broken url.